### DISCUSSION TITLE
What state is this project in?

### DISCUSSION THREAD

**Discussion by lukvermeulen** [rwf2/Rocket#2081] [General]
Really liking this project so far, I’d love to use this in more projects. However I cannot ignore the many open PRs and Issues since almost 6 months, together with a missing of new commits. Thus I come here to ask: Does anybody have some insights on the state of this project? What is needed to keep this growing and moving forward? 

Thanks!

**Comment by dvic** [comment]
there is a long discussion here: https://github.com/SergioBenitez/Rocket/discussions/2011
